internal/cfgplugins/policyforwarding.go

```diff
@@ -508,6 +516,10 @@ func DecapGroupConfigGue(t *testing.T, dut *ondatra.DUTDevice, pf *oc.NetworkIns
 		switch dut.Vendor() {
 		case ondatra.ARISTA:
 			helpers.GnmiCLIConfig(t, dut, decapGroupGUEArista)
+			if ocPFParams.Dynamic {
+				t.Logf("Going into decap")
+				GueDecapConfig(t, dut, ocPFParams)
+			}
```

Comment:
Aren't these mutually exclusive?  Shouldn't this be

```suggestion
			if ocPFParams.Dynamic {
				t.Logf("Going into decap")
				GueDecapConfig(t, dut, ocPFParams)
			}
			else {
				helpers.GnmiCLIConfig(t, dut, decapGroupGUEArista)
			}
```

Reply:
Addressed this comment